src/Illuminate/Console/GeneratorCommand.php

```diff
@@ -70,7 +68,7 @@ public function fire()
 	 */
 	protected function getPath($name)
 	{
-		$name = str_replace($this->getAppNamespace(), '', $name);
+		$name = str_replace(app()->getNamespace(), '', $name);
```

Comment:
I don't really like seeing helper functions for ioc stuff in the core. Feels really hacky.